lib/logger.js

```diff
@@ -67,6 +90,15 @@ class Logger {
   clearContext() {
     this.context = {};
   }
+
+  enableCallStack(bool = true) {
```

Comment:
`enableCallStack` with an argument seems confusing to me. This might be better as a property of the Logger class, with `get` and `set` functions which validate the boolean value (similar to how the `level` property is defined on Logger). This would also avoid having to cache Loggers by setting the `useCallStack` value at the category level (like how the log level is set at the category level).

Reply:
But, that way will influence after new Logger(), not respect to the configuration

Reply:
Take a look at how the `level` property is defined - it always looks up the level in the category configuration, so it applies to all loggers.

Reply:
yup, applies to all same category loggers.